tests/monitoring_test.go

```diff
@@ -287,7 +286,7 @@ var _ = Describe("Prometheus Alerts", func() {
 
 			Eventually(func(g Gomega) error {
 				return apiClient.Get(ctx, types.NamespacedName{Name: vmName, Namespace: strategy.GetNamespace()}, vm)
-			}).Should(MatchError(k8serrors.IsNotFound))
```

Comment:
IIRC this syntax is just missing a string as second argument

e.g.

`Should(MatchError(k8serrors.IsNotFound, "IsNotFound"))`

Reply:
that is what we have in main branch, but that caused the automatic cherry-pick to fail. Probably here we have an older version of gomega

Reply:
ACK